app/services/editions/publishing/events/package.scala

```diff
@@ -1,10 +1,11 @@
 package services.editions.publishing
 
+import model.editions.IssueVersionStatus
 import play.api.libs.json.{Format, Json}
 
 package object events {
 
-  case class PublishEvent(status: String, message: String)
+  case class PublishEvent(status: IssueVersionStatus, message: String)
 
```

Comment:
PublishEvent will need `version` and `timestamp` now as well, but that can be done in next PR

current example JSON:

`\"edition\":\"daily-edition\",\"version\":\"2019-10-02T13:44:18.393Z\",\"issueDate\":\"2019-09-23\",\"status\":\"Processing\",\"message\":\"Publication stage: started\",\"timestamp\":\"2019-10-02T13:44:22+00:00\"`

fields like `edition` and `issueDate` are nice to have i guess

Reply:
Yes, let's push this to the next PR.